apps/accounts/end_user_signup_forms.py

```diff
@@ -167,7 +170,7 @@ def save(self):
         # TODO: Add Crosswalk Create.
         Crosswalk.objects.create(user=new_user,
                                  user_id_hash=binascii.hexlify(pbkdf2(self.cleaned_data['id_number'],
-                                                                      settings.USER_ID_SALT,
+                                                                      get_user_id_salt(settings.USER_ID_SALT),
```

Comment:
Coppied from other PR: change usage to just `get_user_id_salt` so it looks like the functions in `oauth2_provider`